lib/src/participant/local.dart

```diff
@@ -240,6 +243,21 @@ class LocalParticipant extends Participant<LocalTrackPublication> {
     await pub.dispose();
   }
 
+  Future<void> rePublishAllTracks({bool notify = true}) async {
+    final tracks = trackPublications.values.toList();
+    for (LocalTrackPublication track in tracks) {
+      await unpublishTrack(track.sid, notify: notify, stopOnUnpublish: false);
```

Comment:
this might not be needed, because the old server/peer connection is gone.

we do, however, need to republish it to the new room.